src/Features/CSharp/Portable/ConvertToInterpolatedString/CSharpConvertConcatenationToInterpolatedStringRefactoringProvider.cs

```diff
@@ -32,5 +34,17 @@ protected override string GetTextWithoutQuotes(string text, bool isVerbatim, boo
             => isVerbatim
                 ? text.Substring("@'".Length, text.Length - "@''".Length)
                 : text.Substring("'".Length, text.Length - "''".Length);
+
+        protected override bool IsConstMemberDeclaration(SyntaxNode node, ISyntaxFactsService syntaxFacts)
+        {
+            var declaration = node.FirstAncestorOrSelf<MemberDeclarationSyntax>();
```

Comment:
This code can all move into the base class.  either by using generic type params.  or by using ISyntaxFactsService.